pkgs/tools/typesetting/tex/texlive/bin.nix

```diff
@@ -271,10 +264,13 @@ dvipng = stdenv.mkDerivation {
 
   inherit (common) src;
 
-  nativeBuildInputs = [ pkgconfig ];
+  nativeBuildInputs = [ perl pkgconfig ];
   buildInputs = [ core/*kpathsea*/ zlib libpng freetype gd ghostscript makeWrapper ];
 
-  preConfigure = "cd texk/dvipng";
+  preConfigure = ''
+    cd texk/dvipng
+    patchShebangs .
```

Comment:
@jtojnar I thought it was exactly the opposite: there can be future new files that will need to be patched. Can you provide an example of script shebang that need to remain unpatched?

Reply:
Well, if there are new files in need of patching, the build will fail, letting us know. If we patch files that need to be patched with a host path with a build path, it will break at run time and we will not know until someone runs it.